velox/docs/functions/presto/binary.rst

```diff
@@ -8,7 +8,28 @@ Binary Functions
 
 .. function:: from_base64(string) -> varbinary
 
-    Decodes binary data from the base64 encoded ``string``.
+    Decodes a Base64 encoded ``string`` into its original binary representation. 
+    This function is capable of handling both padded and non-padded Base64 encoded strings.
+
+    Examples
+    --------
+    Query with padded Base64 string:
+    ::
+        SELECT from_base64('SGVsbG8gV29ybGQ=');
+
+    Result:
+    ::
+        [72, 101, 108, 108, 111, 32, 87, 111, 114, 108, 100]
+
+    Query with non-padded Base64 string:
+    ::
+        SELECT from_base64('SGVsbG8gV29ybGQ');
+
+    Result:
+    ::
+        [72, 101, 108, 108, 111, 32, 87, 111, 114, 108, 100]
+
+    In these examples, both the padded and non-padded Base64 strings 'SGVsbG8gV29ybGQ=' and 'SGVsbG8gV29ybGQ' decode to the binary representation of the text 'Hello World'.
```

Comment:
This is a very nice description. Typically, in examples, the results are includes as comments

```
SELECT from_base64('SGVsbG8gV29ybGQ='); -- [72, 101, 108, 108, 111, 32, 87, 111, 114, 108, 100]
```

It would be nice to also mention that "partial" padding is supported as well, i.e. if full padding requires ==, a single = is also allowed.

Reply:
Updated it accordingly

Reply:
Thanks. What do you think about

> It would be nice to also mention that "partial" padding is supported as well, i.e. if full padding requires ==, a single = is also allowed.

?

Reply:
@mbasmanova partial padding is not supported.
For example, `YQ==` and `YQ` decode to 'a', while `YQ=`  throws an error.

Reply:
@Joe-Abraham Got it. Thank you for clarifying. Let's mention this in the doc as well for completeness. BTW, does this behavior match Presto?

Reply:
@mbasmanova The behaviour is exactly like presto. I have updated the documentation

![image](https://github.com/facebookincubator/velox/assets/53977252/79c52096-a14d-435b-b321-fef74f453090)